governance/xc_admin/Dockerfile

```diff
@@ -1,18 +0,0 @@
-# Defined in Dockerfile.lerna
```

Comment:
@guibescos fyi I moved this to `governance/xc_admin/packages/crank_executor/Dockerfile` which seemed like the more correct home

Reply:
Hmmmm this one is a bit tricky. crank-executor, crank-pythnet-relayer and proposer-server are all 3 different services that use this image. Is there a way to keep them all three?

Reply:
I think it's good to just have them as a multi service docker in `/svc/`?

Reply:
As a pattern I generally prefer to avoid multi-service images for 2 reasons:

1. It means the image isn't optimized for any specific use case, which is one of the main superpowers of containerization in the first place
2. Doing things with multi-service images is a crutch that folks can use to avoid building good tooling to make it easy and cheap to spin up purpose-built images.  I don't think that crutch is good in the long run and I'd rather feel the pain of not having those tools and be compelled to add tools to make it easier.

That's my opinion, thoughts?